cmd/pilot-discovery/main.go

```diff
@@ -193,8 +193,9 @@ func init() {
 		fmt.Sprintf("File name for Istio mesh configuration"))
 	discoveryCmd.PersistentFlags().StringVarP(&flags.controllerOptions.Namespace, "namespace", "n", "",
 		"Select a namespace for the controller loop. If not set, uses ${POD_NAMESPACE} environment variable")
-	discoveryCmd.PersistentFlags().StringVarP(&flags.controllerOptions.AppNamespace, "app namespace", "a", "",
-		"Restrict the applications namespace that controller manages, do n")
+	discoveryCmd.PersistentFlags().StringVarP(&flags.controllerOptions.WatchedNamespace, "app namespace",
```

Comment:
why have both `a` and `n` flags that seemingly do same thing?

Reply:
They do not control the same thing.

Reply:
Example where they are different is ingress. This PR cleans up the misconception they are the same.
-n was already associated with POD_NAMESPACE, and the command line, and in general associated with the namespace where the k8s object is deployed, so I chose -a to restrict the namespace.